src/app/components/AlwaysEnabledButton/AlwaysEnabledButton.tsx

```diff
@@ -0,0 +1,24 @@
+import { ConditionalTooltip, TooltipConditionPair } from '@/app/components/ConditionalTooltip'
```

Comment:
I don't think that a new button is the solution. We need to update the current button.

Reply:
that would directly affect the dao team as well